session_items.py

```diff
@@ -63,7 +67,37 @@ def save_item(item):
     """
     existing_items = get_items()
     updated_items = [item if item['id'] == existing_item['id'] else existing_item for existing_item in existing_items]
-
+    updated_items= sorted(updated_items,key=lambda d: (d['status'],d['title']))
     session['items'] = updated_items
 
     return item
+
+def clear_items():
+    _DEFAULT_ITEMS_temp = [
+        { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
+        { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
+    ]
+    
+    _DEFAULT_ITEMS_temp= sorted(_DEFAULT_ITEMS_temp,key=lambda d: (d['status'],d['title']))
+    session['items']=_DEFAULT_ITEMS_temp
+    return _DEFAULT_ITEMS_temp
```

Comment:
See my other comment about naming.

But here, it is also a big over-complication.
- You could jump straight to `_DEFAULT_ITEMS_temp = sorted(_DEFAULT_ITEMS, key=...)` since it looks like you want to go back to the original "default items". 
- This function could just be one line clearing the 'items' element from the `session` dictionary - `session.pop('items', None)`